docker_test.go

```diff
@@ -1191,34 +1190,24 @@ func TestEntrypoint(t *testing.T) {
 
 func TestReadTCPropsFile(t *testing.T) {
 	t.Run("HOME is not set", func(t *testing.T) {
-		oldHome := os.Getenv("HOME")
-		os.Unsetenv("HOME")
-		defer func() {
-			os.Setenv("HOME", oldHome)
```

Comment:
Is it required to avoid resetting the env to previous value of the HOME variable?

Reply:
Nope, according to [the docs](https://pkg.go.dev/testing#T.Setenv) that's automatically done after the test is completed

Reply:
I just saw this is brand new in Go 1.17 :sweat_smile: shall I revert it or is it okay to update the test environment?
I don't expect this to break compatibility for the library itself but I'm not sure about it?

Reply:
I replaced the 'fancy' standard lib calls with `gotest.tools` pendants to get compatibility with Go 1.14 again. The dependency was already in place and this way I still can remove filet and the env is automatically managed :blush: